sycl/include/sycl/detail/pi.h

```diff
@@ -1118,9 +1126,15 @@ __SYCL_EXPORT pi_result piextContextCreateWithNativeHandle(
 //
 // Queue
 //
+
+// TODO: Remove during next ABI break and rename piQueueCreateEx to
+// piQueueCreate.
 __SYCL_EXPORT pi_result piQueueCreate(pi_context context, pi_device device,
                                       pi_queue_properties properties,
                                       pi_queue *queue);
+__SYCL_EXPORT pi_result piQueueCreateEx(pi_context context, pi_device device,
```

Comment:
```suggestion
__SYCL_EXPORT pi_result piextQueueCreate(pi_context context, pi_device device,
```
to follow other existing extensions